src/gpl/src/gpuSolver.h

```diff
@@ -0,0 +1,63 @@
+#ifndef __GPUSOLVER__
+#define __GPUSOLVER__
+
+#include <cuda_runtime.h>
```

Comment:
Does this compile if cuda isn't installled?

Reply:
Yes. If there is no cuda installed, gpuSolver.h won't be compiled. The control sentence is written in solver.h.